classes/class-wpcom-liveblog-entry-query.php

```diff
@@ -142,11 +142,6 @@ public static function remove_replaced_deleted_entries( $entries = array(), $num
 				unset( $entries_by_id[ $id ] );
 			}
 
-			// Remove deleted entries
-			if ( 'delete' === $entry->get_type() ) {
-				unset( $entries_by_id[ $id ] );
-			}
-
```

Comment:
I'm a bit worried about this change because this function is already kinda broken. See my changes in [fix/load-more-entires-button](https://github.com/Automattic/liveblog/commits/fix/load-more-entries-button)